analysis/Thematic_Anlaysis.py

```diff
@@ -0,0 +1,103 @@
+# TODO organize imports in separate files
+from analysis.Analysis import Analysis
+from nltk.corpus import stopwords
+from textblob import Word, TextBlob
+from sklearn.decomposition import NMF
+from sklearn.feature_extraction.text import TfidfVectorizer
+import nltk
+
+class Thematic_Analysis(Analysis):
```

Comment:
We will probably have to do some adjusting to the code in these methods. Specifically the pre_process() method to take the user input the same way it's being taken and pre-processed for the sentiment analysis. Or it could go the other way around. I just think both should read the user's responses the same way.

Also, let's keep us-04-thamtic-model/thematicAnalysisMain.py until we know we have everything from there needed.